### PR TITLE
Inverted SQL quotes for MySQL 8.0 compatiblity

### PR DESCRIPTION
**Pull request by rafsuntaskin**
[ET-1021]

Screen Shot 2021-02-24 at 11 49 27 PM

[ET-1021]: https://theeventscalendar.atlassian.net/browse/ET-1021

### CODE REVIEW

File: src/Tribe/Attendee_Repository.php [hunk 1]
```diff
@@ -297,7 +297,7 @@ public function filter_by_optout( $optout ) {
 			case 'no_or_none':
 				$optout_keys = $this->attendee_optout_keys();
 				$optout_keys = array_map( [ $wpdb, '_real_escape' ], $optout_keys );
-				$optout_keys = '"' . implode( '", "', $optout_keys ) . '"';
+				$optout_keys = "'" . implode( '", "', $optout_keys ) . "'";
```
Comment (sc0ttkclark) [inline src/Tribe/Attendee_Repository.php:300]: Double quotes were still used here.

```suggestion
				$optout_keys = "'" . implode( "', '", $optout_keys ) . "'";
```

File: src/Tribe/Repositories/Traits/Post_Attendees.php [hunk 2]
```diff
@@ -69,11 +69,11 @@ public function filter_by_has_attendees( $has_attendees = true ) {
 
 		$event_meta_keys = method_exists( $this, 'attendee_to_event_keys' ) ? $this->attendee_to_event_keys() : [];
 		$event_meta_keys = array_map( [ $wpdb, '_real_escape' ], $event_meta_keys );
-		$event_meta_keys = '"' . implode( '", "', $event_meta_keys ) . '"';
+		$event_meta_keys = "'" . implode( '", "', $event_meta_keys ) . "'";
```
Comment (sc0ttkclark) [inline src/Tribe/Repositories/Traits/Post_Attendees.php:72]: Fix double quotes in the implode

File: src/Tribe/Repositories/Traits/Post_Attendees.php [hunk 3]
```diff
@@ -69,11 +69,11 @@ public function filter_by_has_attendees( $has_attendees = true ) {
...
 
 		$attendee_types = method_exists( $this, 'attendee_types' ) ? $this->attendee_types() : [];
 		$attendee_types = array_map( [ $wpdb, '_real_escape' ], $attendee_types );
-		$attendee_types = '"' . implode( '", "', $attendee_types ) . '"';
+		$attendee_types = "'" . implode( '", "', $attendee_types ) . "'";
```
Comment (sc0ttkclark) [inline src/Tribe/Repositories/Traits/Post_Attendees.php:76]: Fix double quotes in the implode

File: src/Tribe/Repositories/Traits/Post_Attendees.php [hunk 4]
```diff
@@ -140,11 +140,11 @@ public function filter_by_attendee_not_in( $attendee_ids ) {
 
 		$event_meta_keys = method_exists( $this, 'attendee_to_event_keys' ) ? $this->attendee_to_event_keys() : [];
 		$event_meta_keys = array_map( [ $wpdb, '_real_escape' ], $event_meta_keys );
-		$event_meta_keys = '"' . implode( '", "', $event_meta_keys ) . '"';
+		$event_meta_keys = "'" . implode( '", "', $event_meta_keys ) . "'";
```
Comment (sc0ttkclark) [inline src/Tribe/Repositories/Traits/Post_Attendees.php:143]: Fix double quotes in the implode

File: src/Tribe/Repositories/Traits/Post_Attendees.php [hunk 5]
```diff
@@ -140,11 +140,11 @@ public function filter_by_attendee_not_in( $attendee_ids ) {
...
 
 		$attendee_types = method_exists( $this, 'attendee_types' ) ? $this->attendee_types() : [];
 		$attendee_types = array_map( [ $wpdb, '_real_escape' ], $attendee_types );
-		$attendee_types = '"' . implode( '", "', $attendee_types ) . '"';
+		$attendee_types = "'" . implode( '", "', $attendee_types ) . "'";
```
Comment (sc0ttkclark) [inline src/Tribe/Repositories/Traits/Post_Attendees.php:147]: Fix double quotes in the implode

File: src/Tribe/Repositories/Traits/Post_Attendees.php [hunk 6]
```diff
@@ -190,7 +190,7 @@ public function filter_by_attendee_user( $user_ids ) {
 
 		$event_meta_keys = method_exists( $this, 'attendee_to_event_keys' ) ? $this->attendee_to_event_keys() : [];
 		$event_meta_keys = array_map( [ $wpdb, '_real_escape' ], $event_meta_keys );
-		$event_meta_keys = '"' . implode( '", "', $event_meta_keys ) . '"';
+		$event_meta_keys = "'" . implode( '", "', $event_meta_keys ) . "'";
```
Comment (sc0ttkclark) [inline src/Tribe/Repositories/Traits/Post_Attendees.php:193]: Fix double quotes in the implode

File: src/Tribe/Repositories/Traits/Post_Attendees.php [hunk 7]
```diff
@@ -241,7 +241,7 @@ public function filter_by_attendee_user_not_in( $user_ids ) {
...
```
Comment (sc0ttkclark) [inline src/Tribe/Repositories/Traits/Post_Attendees.php:244]: Fix double quotes in the implode

**Review by sc0ttkclark — CHANGES_REQUESTED** [review]
More double quotes still need to be replaced, specifically in the implode() usage and then we need to add a changelog entry

**Comment by rafsuntaskin** [comment]
Updated the remaining inverted quotes within `implode` and added Changelog.

**Review by sc0ttkclark — APPROVED** [review]

**Pull request merged**